kong/db/strategies/off/init.lua

```diff
@@ -213,6 +214,15 @@ local function select_by_key(schema, key)
 
   entity = schema:process_auto_fields(entity, "select", true, PROCESS_AUTO_FIELDS_OPTS)
 
+  if entity and entity.ttl then
+    local ttl_value = entity.ttl - now()
+    if ttl_value > 0 then
```

Comment:
It is relative simple, could we change it to lua's `and or`, like
```
entity.ttl = ttl_value > 0 and ttl_value  or 0
```

Reply:
done